host/2.0/stretch/amd64/appservice/python.Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG BASE_IMAGE=mcr.microsoft.com/azure-functions/python:2.0
+FROM ${BASE_IMAGE}
+
+EXPOSE 2222 80
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends openssh-server dialog && \
+    echo "root:Docker!" | chpasswd
```

Comment:
You can remove the user name and password now, with the latest az webapp ssh command, there is no need for the user/password set in the container.

Reply:
does that also work for remote debugging and kudu ssh?

Reply:
You need the password - az webapp ssh hides the username password from the customer but its still required.